### DISCUSSION TITLE
Make name of 'collides' boolean

### DISCUSSION THREAD

**Discussion by neu5** [Annoraaq/grid-engine#60] [Ideas]
Currently the name of custom property for detecting collisions is hardcoded as 'collides'.
I was wondering isn't it would be better to have it 'collides' as default but changeable through the config?

**Comment by Annoraaq** [comment]
I thought about this, too. Newer properties on the tilemap have a "gm_" prefix which would make sense as a default for 'collides', too.

What do you think about the following two changes:

1. change default from 'collides' to 'gm_collides' (and keep 'collides' for backwards compatibility)
2. make the name of the property configurable

**Reply by neu5** [reply]
I think it totally makes sense. You might also drop the backwards compatibility part as of 2.0 version

**Reply by Annoraaq** [reply]
@neu5 It is live with version 1.9.0.

The new default is `gm_collide` and you can set a custom name in the config for the plugin:

https://github.com/Annoraaq/phaser-grid-movement-plugin/wiki/Config

**Reply by neu5** [reply]
@Annoraaq cool! You definitely don't waste time :)
Btw. do you have any special plans for version 2.0?
Is it going to be 2.0 with the next new feature?

**Reply by Annoraaq** [reply]
Hey @neu5. Currently I don't have any plans for 2.0 in sight. 

In case you are asking because we are at 1.9: it will just continue counting on (1.10, 1.11, etc).

There will only be a version 2.0 if there are major breaking changes.
